test/Microsoft.ML.StaticPipelineTesting/Training.cs

```diff
@@ -121,8 +121,7 @@ public void SdcaBinaryClassification()
             var est = reader.MakeNewEstimator()
                 .Append(r => (r.label, preds: ctx.Trainers.Sdca(r.label, r.features,
                     maxIterations: 2,
-                    onFit: (p, c) => { pred = p; cali = c; },
-                    advancedSettings: s => s.NumThreads = 1)));
```

Comment:
> s.NumThreads = 1 [](start = 42, length = 17)

And here.

Reply:
fixed

---
In reply to: [248104248](https://github.com/dotnet/machinelearning/pull/2093#discussion_r248104248) [](ancestors = 248104248)